<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>811</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_root">
         <attribute name="title">
          <string>Rooteo</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>20</y>
            <width>271</width>
            <height>681</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_rooteo">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>Buscar Widget ...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>267</width>
                <height>585</height>
               </rect>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>251</width>
                 <height>621</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>todo Foto</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>insertar Led</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="text">
                   <string>insertar Relay</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>todo Foto</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>todo Foto</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>todo Foto</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>insertar Display</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="text">
                   <string>insertar pulsador</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Cargar PCB Existente</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_10">
               <property name="text">
                <string>Exportar PCB</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>Iniciar Rooteo / Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_config">
         <attribute name="title">
          <string>Configuraci√≥n</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>271</width>
            <height>671</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_eventos">
             <property name="text">
              <string>Eventos</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Evento en .umf</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Widget Asociado</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollEventos">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>267</width>
                <height>254</height>
               </rect>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>13</y>
                 <width>251</width>
                 <height>251</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Evento_1</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Evento_2</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Evento_5</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Evento_4</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Evento_3</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBox_2"/>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="comboBox_3"/>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="comboBox_4"/>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="comboBox_5"/>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_acciones">
             <property name="text">
              <string>Acciones</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Acci√≥n en .umf</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Widget Asociado</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollAcciones">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>267</width>
                <height>253</height>
               </rect>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>251</width>
                 <height>271</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="formLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Accion_1</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBox_6"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Accion_2</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBox_7"/>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_11">
             <property name="text">
              <string>Guardar Proyecto</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Cargar .umf</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QMdiArea" name="mdiArea">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuNuevo">
     <property name="title">
      <string>Nuevo</string>
     </property>
     <addaction name="actionNuevoProyectoUMH"/>
     <addaction name="actionNuevoPCB"/>
    </widget>
    <widget class="QMenu" name="menuGuardar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Guardar Como</string>
     </property>
     <addaction name="actionGuardarProyectoUMH"/>
     <addaction name="actionGuardarPCB"/>
    </widget>
    <widget class="QMenu" name="menuLimpiar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Limpiar</string>
     </property>
     <addaction name="actionLimpiarPCB"/>
     <addaction name="actionLimpiarUMF"/>
    </widget>
    <widget class="QMenu" name="menuAbrir">
     <property name="title">
      <string>Abrir</string>
     </property>
     <addaction name="actionAbrirPCB"/>
     <addaction name="actionAbrirUMF"/>
     <addaction name="actionAbrirProyectoUMH"/>
    </widget>
    <addaction name="menuNuevo"/>
    <addaction name="menuAbrir"/>
    <addaction name="menuGuardar"/>
    <addaction name="separator"/>
    <addaction name="menuLimpiar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionInstrucciones"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuPerspectiva">
    <property name="title">
     <string>Perspectiva</string>
    </property>
    <addaction name="actionRooteo"/>
    <addaction name="actionConfiguraci_n"/>
    <addaction name="actionSimulaci_n"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuPerspectiva"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConfigurarServidorToolbar"/>
   <addaction name="actionIniciarSimulacionToolbar"/>
   <addaction name="actionGuardarUMHToolbar"/>
  </widget>
  <action name="actionNuevoProyectoUMH">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Proyecto UMH</string>
   </property>
  </action>
  <action name="actionNuevoPCB">
   <property name="text">
    <string>PCB</string>
   </property>
  </action>
  <action name="actionGuardarProyectoUMH">
   <property name="text">
    <string>Proyecto UMH</string>
   </property>
  </action>
  <action name="actionGuardarPCB">
   <property name="text">
    <string>PCB</string>
   </property>
  </action>
  <action name="actionLimpiarPCB">
   <property name="text">
    <string>PCB</string>
   </property>
  </action>
  <action name="actionLimpiarUMF">
   <property name="text">
    <string>UMF</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Abrir Proyecto UMH</string>
   </property>
  </action>
  <action name="actionAbrirProyectoUMH">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Proyecto UMH</string>
   </property>
  </action>
  <action name="actionAbrirPCB">
   <property name="text">
    <string>PCB</string>
   </property>
  </action>
  <action name="actionAbrirUMF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>UMF</string>
   </property>
  </action>
  <action name="actionInstrucciones">
   <property name="text">
    <string>Instrucciones</string>
   </property>
  </action>
  <action name="actionRooteo">
   <property name="text">
    <string>Rooteo</string>
   </property>
  </action>
  <action name="actionConfiguraci_n">
   <property name="text">
    <string>Configuraci√≥n</string>
   </property>
  </action>
  <action name="actionSimulaci_n">
   <property name="text">
    <string>Simulaci√≥n</string>
   </property>
  </action>
  <action name="actionIniciarSimulacionToolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/play-button.png</normaloff>:/images/play-button.png</iconset>
   </property>
   <property name="text">
    <string>Iniciar Simulacion</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionConfigurarServidorToolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configurar Servidor</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionGuardarUMHToolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/diskette.png</normaloff>:/images/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Guardar UMH</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDebug">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
